docusaurus/docs/get-started/best-practices.md

```diff
@@ -85,6 +85,27 @@ Is something missing from this list? [Let us know](https://github.com/grafana/pl
 
 In the Grafana plugin ecosystem, plugin compatibility with specific Grafana versions is determined by the semantic versioning range defined in the `grafanaDependency` property of the plugin's `plugin.json` file. Plugin authors must carefully select a version range that balances broad compatibility with the need for manageable maintenance efforts.
 
+:::note
+
+We strongly recommend to verify that the range you have defined in `grafanaDependency` is set correctly. The best way to verify this is using [`semver.satisfies`](https://www.npmjs.com/package/semver) function to check a specific version against the range you have defined.
+
+Example - your `plugin.json` defines `grafanaDependency: >=12.1.0`. In order to test if it would be installable on, for example, Grafana version `12.1.0-3212123` you can verify it with:
+
+```
+console.log(semver.satisfies("12.1.0-3212123", ">=12.1.0"));
+> false
+```
+
+As you see it would not be installable. If you want to target this version you can adjust your `grafanaDependency` to `>=12.1.0-0`
+
+```
+console.log(semver.satisfies("12.1.0-3212123", ">=12.1.0-0"));
+> true
+```
+
+Read more on semver specifications on [semver.org](https://semver.org)
+:::
```

Comment:
this is not being rendered as you expect, I think you need to add a blank line before the `:::` (check the "rich diff" view)

Reply:
it is rendering correctly. Unfortunately rich diff != docusaurus rendering. It is close to but not the same.